honeybee/model.py

```diff
@@ -119,243 +207,391 @@ def north_vector(self, value):
     @property
     def rooms(self):
         """A list of all Room objects in the model."""
-        return self._rooms
+        return tuple(self._rooms)
 
     @property
     def faces(self):
         """A list of all Face objects in the model."""
-        return self._faces
+        child_faces = []
+        for room in self._rooms:
```

Comment:
List comprehension is most likely faster for a case like this if we have so many rooms

Reply:
The only reason why I did this previously was because I needed to use `extend` instead of append.  But now I see that it is possible to do stacked list comprehensions like this:
```
[face for room in self._rooms for face in room._faces]
```